docs/hugo/content/guide/crd-management.md

```diff
@@ -0,0 +1,121 @@
+
+Example: `resources.azure.com/*;containerservice.azure.com/*;keyvault.azure.com/*;managedidentity.azure.com/*;eventhub.azure.com/*`
+
+> **Note:** If you are specifying the Helm values from a shell with `--set crdPattern=...` you must wrap the crdPattern
+> argument in single quotes `'` if it contains wildcard characters the shell would expand. For example:
+> `--set crdPattern='resources.azure.com/*;containerservice.azure.com/*'`
+
+### Manual CRD installation
+
+If you do not want to grant the operator permission to create + update CRDs, you can instead manage the CRDs yourself.
+To do this you can use the customresourcedefinitions YAML that is part of each release. This YAML contains just the ASO CRDs.
+You can then filter to whichever subset of CRDs you want or apply them all. 
+See for example [azureserviceoperator_customresourcedefinitions_v2.0.0.yaml](https://github.com/Azure/azure-service-operator/releases/download/v2.0.0/azureserviceoperator_customresourcedefinitions_v2.0.0.yaml)
+
+The process to do this for a fresh install is:
+1. Install the CRDs you want to use. These must be the CRDs for the same version of the operator you install in step 2.
+2. Install the operator Helm chart. Specify `installCRDs: false` as part of `values.yaml` when installing.
+3. Ensure the pod launches correctly.
+
+The process for upgrading the operator is similar:
+1. Upgrade the CRDs you are using. These must be the CRDs for the same version of the operator you upgrade to in step 2.
+2. Upgrade the operator Helm chart. Specify `installCRDs: false` as part of `values.yaml` when upgrading.
+3. Ensure the pod launches correctly.
+
+## With YAML
+
+### Automatic CRD installation (recommended)
+
+Unlike Helm there are no parameters built into the raw YAML. You must use a tool such as `kustomize` or `sed` to
+set the `--crd-pattern` command-line argument of the operator pod to include the CRDs you wish to install.
+
+Example: `cat operator.yaml | sed 's_--crd-pattern=.*_--crd-pattern=resources.azure.com/*;containerservice.azure.com/*_g' | kubectl apply -`
+
+Alternatively you can use `kustomize` with a configuration like:
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - https://github.com/Azure/azure-service-operator/releases/download/v2.1.0/azureserviceoperator_v2.1.0.yaml
+
+patches:
+  - patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/args/-
+        value: --crd-pattern=resources.azure.com/*;containerservice.azure.com/*;keyvault.azure.com/*;managedidentity.azure.com/*;eventhub.azure.com/*
+    target:
+      kind: Deployment
+```
+
+### Manual CRD installation
+
+If you do not want to grant the operator permission to create + update CRDs, you can instead manage the CRDs yourself.
+To do this you can use the customresourcedefinitions YAML that is part of each release. This YAML contains just the ASO CRDs.
+You can then filter to whichever subset of CRDs you want or apply them all.
```

Comment:
```suggestion
If you do not want to grant the operator permission to create + update CRDs, you can instead manage the CRDs yourself.
To do this you can filter a subset of desired CRDs from the YAML file containing all the ASO custom resource definitions that we include with each release.
```

Reply:
Changed to:
> To do this you can use a YAML file containing all the ASO custom resource definitions that we include with each release. 
This file contains _only_ the ASO CRDs.

Which is what the above version of this section had.